src/components/LoginCard/index.jsx

```diff
@@ -1,32 +1,54 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
 
-// import { auth, googleProvider } from 'services/FirebaseService';
 import { Button, Card, Form, Input, Space, Typography } from 'antd';
 
 import { GoogleCircleFilled, LockOutlined, UserOutlined } from '@ant-design/icons';
 
 import './LoginCard.scss';
+import { useAuth } from 'services/AuthProvider';
+import { googleProvider } from 'services/FirebaseService';
 
 const { Text } = Typography;
 
 function LoginCard() {
-  // const googleSignIn = async () => {
-  //   try {
-  //     const response = await auth.signInWithPopup(googleProvider);
-  //     const { user } = response.user;
-  //     // eslint-disable-next-line
-  //     console.log('user object', user);
-  //   } catch (error) {
-  //     // eslint-disable-next-line
-  //     console.error(error.message);
-  //   }
-  // };
+  const { emailLogin, googleLogin } = useAuth();
+  // eslint-disable-next-line no-unused-vars
+  const [loading, setLoading] = useState(false);
+  const history = useHistory();
+
+  async function handleSubmit(userInput) {
+    try {
+      /* CHANGE LOADER */
+      setLoading(true);
+      await emailLogin(userInput.email, userInput.password);
+      setLoading(false);
+      return history.push('/');
+    } catch (errors) {
+      /* FOR TESTING (NEEDS IMPLEMENTATION OF ERROR HANDLING) */
+      // eslint-disable-next-line no-console
+      return console.log(errors.message);
+    }
+  }
```

Comment:
Why does handle submit return something? Isn't this suppose to only be a side effect? i.e Only calling the emails and redirecting users.

Also for using routes when pushing, use constants
```js
// from
history.push('/')
// to
history.push(route.ROOT)

```